deploy-service/teletraanservice/src/test/java/com/pinterest/teletraan/fixture/EnvironBeanFixture.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2024 Pinterest, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.pinterest.teletraan.fixture;
+
+import com.pinterest.deployservice.bean.AcceptanceType;
+import com.pinterest.deployservice.bean.EnvironBean;
+import com.pinterest.deployservice.bean.EnvironState;
+import com.pinterest.deployservice.bean.OverridePolicy;
+import java.util.UUID;
+
+public class EnvironBeanFixture {
+    public static EnvironBean createRandomEnvironBean() {
+        EnvironBean environBean = new EnvironBean();
+        environBean.setEnv_name(UUID.randomUUID().toString());
+        environBean.setStage_name(UUID.randomUUID().toString());
+        environBean.setEnv_id(UUID.randomUUID().toString());
+        environBean.setDeploy_id(UUID.randomUUID().toString());
+        environBean.setState(EnvironState.NORMAL);
+        environBean.setSuccess_th(10000);
```

Comment:
Nice approach with fixtures 👍

But let's make a fixture fully random to improve quality of tests, let's use `UUID.randomUUID().toString()` where need to populate string value instead of hardcoded values, like `envvar_id_1`. Because hard coded values leads to the flaky tests.